spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -1,1901 +1,2644 @@
+            it 'logs the job completed analytic' do
+              expect(analytics).to have_received(
+                :idv_in_person_usps_proofing_results_job_completed,
+              ).with(
+                **default_job_completion_analytics,
+                enrollments_checked: 1,
+                enrollments_errored: 1,
+                percent_enrollments_errored: 100.0,
+              )
             end
+          end
 
-            it 'updates the timestamp but does not update the status or log a message' do
-              freeze_time do
-                pending_enrollment.update(
-                  enrollment_established_at: Time.zone.now - 3.days,
-                  status_check_attempted_at: Time.zone.now - 1.day,
-                  status_updated_at: Time.zone.now - 1.day,
+          context 'when the USPS proofing results is a hash' do
+            let(:usps_enrollment_start_date) do
+              current_time.getlocal('-0600')
+            end
+
+            let(:usps_enrollment_end_date) do
+              (current_time + in_person_results_delay_in_hours.hour).getlocal('-0600')
+            end
+
+            let(:response_body) do
+              {
+                status: 'In-person passed',
+                proofingPostOffice: 'WILKES BARRE',
+                proofingCity: 'WILKES BARRE',
+                proofingState: 'PA',
+                enrollmentCode: enrollment.enrollment_code,
+                primaryIdType: "State driver's license",
+                transactionStartDateTime:
+                  usps_enrollment_start_date.strftime('%m/%d/%Y %H%M%S'),
+                transactionEndDateTime: usps_enrollment_end_date.strftime('%m/%d/%Y %H%M%S'),
+                fraudSuspected: false,
+                proofingConfirmationNumber: '350040248346701',
+                ippAssuranceLevel: '1.5',
+              }
+            end
+
+            let(:response_analytics) do
+              {
+                fraud_suspected: response_body[:fraudSuspected],
+                primary_id_type: response_body[:primaryIdType],
+                secondary_id_type: response_body[:secondaryIdType],
+                failure_reason: response_body[:failureReason],
+                transaction_end_date_time: usps_enrollment_end_date.getlocal('UTC'),
+                transaction_start_date_time: usps_enrollment_start_date.getlocal('UTC'),
+                status: response_body[:status],
+                assurance_level: response_body[:assuranceLevel],
+                proofing_post_office: response_body[:proofingPostOffice],
+                proofing_city: response_body[:proofingCity],
+                proofing_state: response_body[:proofingState],
+                scan_count: response_body[:scanCount],
+                response_message: response_body[:responseMessage],
+                response_present: true,
+              }
+            end
+
+            context 'when the InPersonEnrollment has fraud results pending' do
```

Comment:
In `handle_successful_status_update` should we ensure we are not calling the methods inside the unless block (unless fraud_results_pending?). You may be doing this and I have not gotten to to. It is in the success path